lib/v2/iwara/subscriptions.js

```diff
@@ -9,39 +9,63 @@ const md = require('markdown-it')({
 
 module.exports = async (ctx) => {
     const rootUrl = `https://www.iwara.tv`;
-    const videoSubUrl = 'https://api.iwara.tv/videos?page=0&limit=30&subscribed=true';
-    const imageSubUrl = 'https://api.iwara.tv/images?page=0&limit=30&subscribed=true';
-    const cookie = config.iwara.cookie;
+    const username = config.iwara.username;
+    const password = config.iwara.password;
 
-    if (cookie === undefined) {
+    if (username === undefined || password === undefined) {
         throw Error('Iwara subscription RSS is disabled due to the lack of <a href="https://docs.rsshub.app/install/#pei-zhi-bu-fen-rss-mo-kuai-pei-zhi">relevant config</a>');
     }
 
+    // get token
+    const loginResponse = await got({
+        method: 'post',
+        url: 'https://api.iwara.tv/user/login',
+        headers: {
+            'content-type': 'application/json',
+        },
+        data: JSON.stringify({
+            email: username,
+            password,
+        }),
+    });
+
+    const authHeaders = {
+        authorization: 'Bearer ' + loginResponse.data.token,
+    };
```

Comment:
Please cache the authorization header using a cache key like `iwara:token` so we don't need to login again.